packages/react/src/themes/teams/components/Label/labelVariables.ts

```diff
@@ -1,9 +1,25 @@
 import { pxToRem } from '../../../../lib'
-import { SiteVariablesPrepared, ColorSchemeMapping } from '../../../types'
-import { extendColorScheme } from '../../../colorUtils'
+import {
+  SiteVariablesPrepared,
+  StrictColorSchemeMapping,
+  StrictColorScheme,
+  ComponentAreaName,
+} from '../../../types'
+import { extendColorScheme, pickValuesFromColorScheme } from '../../../colorUtils'
+import { TeamsColorNames } from '../../colors'
+
+// TODO extract this to a util
+const tuple = <T extends ComponentAreaName>(...args: T[]) => args
```

Comment:
`tuple` is a bit misleading, not sure what this does..

Reply:
ah, I understand now..by looking on next line. pls add JSdoc and example of usage in comments, would be helpful

Reply:
Yep, will add this, thank you!

Reply:
I refactored this to use an util function. This is the current code:
```
const labelColorComponentAreas = generateComponentAreas('foreground', 'background')
type LabelColorComponentAreas = typeof labelColorComponentAreas[number]

type LabelColorSchemeMapping = StrictColorSchemeMapping<
  StrictColorScheme<LabelColorComponentAreas>,
  TeamsColorNames
>
```
Does this make more sense now?